src/index.html

```diff
@@ -5,8 +5,292 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#apple">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#phones">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#computers"
+            >Ноутбуки и компьютеры</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#planshets">Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">Фото</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#video">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="catalog">
+        <div class="catalog__wrap">
+          <div class="product-card" data-qa="card">
+            <img
+              src="./images/imac.jpeg"
+              alt="Product image"
+              class="product-card__image"
+            >
+            <div class="product-card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</div>
+            <div class="product-card__articul">Код товара: 195434</div>
+            <div class="product-card__extra">
+              <div class="product-card__stars">
+                <div class="stars stars--4">
+                  <div class="stars__star star"></div>
```

Comment:
you don't need `star` block, fix it everywhere
```suggestion
                  <div class="stars__star"></div>
```